src/MergeContext.js

```diff
@@ -298,11 +303,13 @@ class MergeContext {
         const collaborators = await GH.getCollaborators();
         const pushCollaborators = collaborators.filter(c => c.permissions.push === true);
         const requestedReviewers = this._prRequestedReviewers();
+        this._approvalResult = {params: null, delayMs: -1};
```

Comment:
I see no reason to introduce a magic numerical constant instead of using a much more descriptive/appropriate `null`. If there are not good reasons for `-1` please rewrite to use `null` instead.